client/src/components/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -31,6 +40,7 @@ const ProfilePhoto = () => {
       setFilename(`Failed to upload image!`);
     }
   };
+>>>>>>> ae795a08e4f75fa7bbf9a612ce4f81bd52d16c94
```

Comment:
Git's internal messages have been committed to the PR, please remove them 😄

Reply:
added only the updated documents this commit

Reply:
Please remove these commit codes before merging